extra/check-lang-json.js

```diff
@@ -0,0 +1,27 @@
+// For #5231
+
+const fs = require("fs");
+
+let path = "./src/lang";
+
+// list directories in the lang directory
+let jsonFileList = fs.readdirSync(path);
+
+for (let jsonFile of jsonFileList) {
+    if (!jsonFile.endsWith(".json")) {
+        continue;
+    }
+
+    let jsonPath = path + "/" + jsonFile;
+    let originalContent = fs.readFileSync(jsonPath, "utf8");
+    let langData = JSON.parse(originalContent);
+
+    let formattedContent = JSON.stringify(langData, null, 4) + "\n";
+
+    if (originalContent !== formattedContent) {
+        console.error(`File ${jsonFile} is not formatted correctly.`);
+        process.exit(1);
+    }
```

Comment:
instead of just aborting here (imagine getting "not correctly formatted"..)
=> lets abort like this in CI and fix the formatting issue locally instead.

Reply:
Yes, it does abort the CI as a non-zero exit code means error in Linux. 
Tested in 2475baa.